css/style.css

```diff
@@ -0,0 +1,15 @@
+#img-div {
+   
+    align-self: center;
+}
+#image {
+    display: block;
+    position: inherit;
+    align-self: center;
+    max-width: 100%;
+    height: auto;
```

Comment:
Don't you think it will be better to use the right selectors in styling your page. Please use (.class-selector) instead of the (#id-selector) in styling your page.

Reply:
- [Optinal] Good job overall @shegy28, You could explain the why here in depth to help the other person understand why you suggest `class selector` instead of `id`.